bin/sqs_cli.py

```diff
@@ -10,41 +10,21 @@
 #
```

Comment:
I moved this script into `bin` where all the scripts should live. Plus it should be able to run outside of `socorro-cmd`.